### PR TITLE
	new file:   data/BLUE/data.json

### PR DESCRIPTION
**Pull request by blueswapbase**
new file:   data/BLUE/logo.svg

**Description**

Adding the BLUE token to the Base token list.
@roberto-bayardo

### CODE REVIEW

File: data/BLUE/data.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "BlueSwap",
+  "symbol": "BLUE",
+  "decimals": 18,
+  "description": "Swap & Liquidity Pool Token for BlueSwap DEX",
+  "website": "https://blueswap.app",
+  "twitter": "@blueswapbase",
+  "tokens": {
```
Comment (roberto-bayardo) [inline data/BLUE/data.json:8]: does this token exist on Ethereum? If so please add the appropriate section here.

Comment (blueswapbase) [reply data/BLUE/data.json:8]: It does not, does it need to exist on Ethereum to be in the list? It's a Base only token.

Comment (roberto-bayardo) [reply data/BLUE/data.json:8]: No it does not, but if it's a native L2 token that can't be bridged you'll want to add the "nobridge": true directive to your data.json.

Comment (blueswapbase) [reply data/BLUE/data.json:9]: Thank you, done I added that to the request.

**Comment by roberto-bayardo** [comment]
Would you mind regenerating the L2 token using this token factory (same factory different address)?

https://basescan.org/address/0xf23d369d7471bd9f6487e198723eea023389f1d4

We discovered that the default factory generates addresses that conflict with different tokens on Optimism network, which we'd prefer to avoid.

**Comment by blueswapbase** [comment]
Is this right?
https://basescan.org/tx/0xf068ab8664b7e09dcad9d5ab2512aef8ed05883fa2e2b309e136170e277a668b

https://basescan.org/tx/0xba3586d45e5d07c775e21f5817daf647947a3a35f31aad1f47a60457f93eb90d

Just did the following:
_remote address (0x30136B90e532141FeD006c61105cff3668b5c774)
_name
blueswap
_symbol
BLUE

**Comment by roberto-bayardo** [comment]
> Is this right? 

So now that I think about this a bit more since this is an L2 native token I don't think it makes sense to deploy using the factory at all.  The "remote address" is supposed to be the address of your token on the L1 (which if your token is native you don't have).   

Can you just deploy a standard ERC20 contract on Base for your token rather than rely on the bridge factory?

**Comment by blueswapbase** [comment]
Sorry this is where I'm confused, the token was initially created only with ERC20 contract only. This is the contract used: 

`// SPDX-License-Identifier: MIT
pragma solidity ^0.8.9;

import "@openzeppelin/contracts@4.9.3/token/ERC20/ERC20.sol";
import "@openzeppelin/contracts@4.9.3/token/ERC20/extensions/ERC20Burnable.sol";
import "@openzeppelin/contracts@4.9.3/token/ERC20/extensions/ERC20Snapshot.sol";
import "@openzeppelin/contracts@4.9.3/access/AccessControl.sol";
import "@openzeppelin/contracts@4.9.3/security/Pausable.sol";
import "@openzeppelin/contracts@4.9.3/token/ERC20/extensions/ERC20Permit.sol";
import "@openzeppelin/contracts@4.9.3/token/ERC20/extensions/ERC20Votes.sol";
import "@openzeppelin/contracts@4.9.3/token/ERC20/extensions/ERC20FlashMint.sol";

contract BlueSwap is ERC20, ERC20Burnable, ERC20Snapshot, AccessControl, Pausable, ERC20Permit, ERC20Votes, ERC20FlashMint {
    bytes32 public constant SNAPSHOT_ROLE = keccak256("SNAPSHOT_ROLE");
    bytes32 public constant PAUSER_ROLE = keccak256("PAUSER_ROLE");
    bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");

    constructor() ERC20("BlueSwap", "BLUE") ERC20Permit("BlueSwap") {
        _grantRole(DEFAULT_ADMIN_ROLE, msg.sender);
        _grantRole(SNAPSHOT_ROLE, msg.sender);
        _grantRole(PAUSER_ROLE, msg.sender);
        _mint(msg.sender, 1000000000 * 10 ** decimals());
        _grantRole(MINTER_ROLE, msg.sender);
    }

    function snapshot() public onlyRole(SNAPSHOT_ROLE) {
        _snapshot();
    }

    function pause() public onlyRole(PAUSER_ROLE) {
        _pause();
    }

    function unpause() public onlyRole(PAUSER_ROLE) {
        _unpause();
    }

    function mint(address to, uint256 amount) public onlyRole(MINTER_ROLE) {
        _mint(to, amount);
    }

    function _beforeTokenTransfer(address from, address to, uint256 amount)
        internal
        whenNotPaused
        override(ERC20, ERC20Snapshot)
    {
        super._beforeTokenTransfer(from, to, amount);
    }

    // The following functions are overrides required by Solidity.

    function _afterTokenTransfer(address from, address to, uint256 amount)
        internal
        override(ERC20, ERC20Votes)
    {
        super._afterTokenTransfer(from, to, amount);
    }

    function _mint(address to, uint256 amount)
        internal
        override(ERC20, ERC20Votes)
    {
        super._mint(to, amount);
    }

    function _burn(address account, uint256 amount)
        internal
        override(ERC20, ERC20Votes)
    {
        super._burn(account, amount);
    }
}
`

**Comment by roberto-bayardo** [comment]
Ahah, apologies then.  In that case the original token should be fine, but you'll have to [verify](https://basescan.org/verifyContract?a=0x30136B90e532141FeD006c61105cff3668b5c774) the contract code in basescan so we can confirm it's standard.

**Comment by blueswapbase** [comment]
Understood, verified! 
https://basescan.org/address/0x30136b90e532141fed006c61105cff3668b5c774#code

**Review by roberto-bayardo — APPROVED** [review]
looks good thanks.

**Review by hamdiallam — APPROVED** [review]

**Pull request merged**